kafka_consumer/tox.ini

```diff
@@ -0,0 +1,86 @@
+
+[tox]
+minversion = 2.0
+basepython = py27
+envlist =
+    kafkaconsumer-{0_9_0_1,0_10_2_1,0_11_0_1}-{kafka,zk}
+    flake8
+
+[common]
+deps =
```

Comment:
Let's put `deps` only in `[testenv]`.

Reply:
This way they'd be installed also in `testenv:flake8`, not a big deal probably but technically not needed

Reply:
Actually no because `deps = flake8` overrides that.